File: Daily Test/C/Count Of Digits In String 14.5.2026.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main() {

    int n, x;
    scanf("%d %d", &n, &x);

    char str[1000];

    for(int i = 0; i < n; i++) {

        scanf("%s", str);

        int cnt = 0;

        for(int j = 0; str[j] != '\0'; j++) {

            if(isdigit(str[j]) && (str[j] - '0') == x) {
                cnt++;
            }
        }

        printf("%d ", cnt);
    }

    return 0;
}

File: Daily Test/Java/Count Of Digits In String 14.5.2026.java
import java.util.*;

public class Hello {

    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        int n = s.nextInt();
        int x = s.nextInt();

        String[] str = new String[n];

        for(int i = 0; i < n; i++){
            str[i] = s.next();
        }

        for(String w : str){

            int cnt = 0;

            for(char ch : w.toCharArray()){

                if(Character.isDigit(ch) && (ch - '0') == x){
                    cnt++;
                }
            }

            System.out.print(cnt + " ");
        }
    }
}